kaldb/src/test/java/com/slack/kaldb/server/KalDbIntegrationTest.java

```diff
@@ -24,10 +28,16 @@
   private static final Logger LOG = LoggerFactory.getLogger(KalDbIntegrationTest.class);
 
   private Kaldb kaldb;
+  private TestingServer testingServer;
+  private EphemeralKafkaBroker broker;
   private final ObjectMapper om = new ObjectMapper();
 
   @Before
-  public void start() throws IOException {
+  public void start() throws Exception {
+    testingServer = new TestingServer(2181);
+    broker = EphemeralKafkaBroker.create(9092);
```

Comment:
Should we use random ports or do you think using standard ports is fine here?

Reply:
These need to be the same as what is provided in the `config/config.yaml`, as that's where we're reading the connection strings from. If we want to use dynamic values we'd need to set the ENV vars, which I felt was less optimal than using the standard ports.

Reply:
Oh, I meant would it be better to use a random port like 3333 instead of 2181. We can update the config also with this information. Not dynamic port numbers from tests.

Reply:
Ah, in that case I think leaving them to the default ports is fine. If they're static there's not a significant advantage to changing them from the defaults.